apps/member-profile/app/shared/components/company-review.tsx

```diff
@@ -47,10 +48,13 @@ type CompanyReviewProps = {
   reviewedAt: string;
   text: string;
   title: string;
+  upvotes?: string | null;
+  upvoted?: boolean | null;
```

Comment:
nit: We can also change the naming here from `upvoted` to `hasUpvoted`, and `upvotes` to `upvotesCount`